spades.wdl

```diff
@@ -0,0 +1,79 @@
+task spades {
+    String outputDir
+    String? preCommand
+    File? read1
+    File? read2
+    File? singleRead
+    File? interlacedReads
+    File? sangerReads
+    File? pacbioReads
+    File? nanoporeReads
+    File? tslrContigs
+    File? trustedContigs
+    File? untrustedContigs
+    Boolean? singleCell
+    Boolean? metagenomic
+    Boolean? rna
+    Boolean? plasmid
+    Boolean? ionTorrent
+    Boolean? onlyErrorCorrection
+    Boolean? onlyAssembler
+    Boolean? careful
+    Boolean? disableGzipOutput
+    Boolean? disableRepeatResolution
+    File? dataset
+    Int threads
+    Int memoryGb
```

Comment:
Should threads and memory really be mandatory? Seems more like a thing to make optional with a default.

Reply:
There were some issues with running spades on the cluster and defining defaults